### DISCUSSION TITLE
Create a basic starter kit with auth

### DISCUSSION THREAD

**Discussion by diegogit03** [adonisjs/core#4124] [Ideas]
I am thinking in Something like [Laravel breeze](https://laravel.com/docs/10.x/starter-kits#laravel-breeze), where we have a "auth" template/starter kit, this is an amazing resource for create quickly new projects with adonisjs.

I  am available for work on this. anyone else interested in this?

**Comment by MaximeMRF** [comment]
Hello @diegogit03, I'm also interested of doing starter kits

**Comment by RomainLanz** [comment]
Hey! 👋🏻 

They are already some available in the community.

- [Portal - Scaffolding package to generate Auth flows](https://github.com/adonisaddons/portal)
- [💰 Pace - Everything you need in a SaaS application](https://adonispace.dev)

It will be easier to create and share boilerplates in V6.

**Reply by diegogit03** [reply]
Okay
